src/components/patient/patientCard/index.tsx

```diff
@@ -90,34 +40,50 @@ const PatientCard = ({ patientId }: Props) => {
     if (!patientData) {
       return (
         <>
-          <Callout title="Patient not found" intent="danger">
+          <Callout title="Donnée non chargées"></Callout>
+        </>
+      );
+    } else if (patientData === undefined) {
```

Comment:
this is already catched by the first "if", `if (!patient)` is the same as ` if (patientData === undefined)`